extras/syntax-highlighting/visual-code/nestml/language-configuration.json

```diff
@@ -0,0 +1,35 @@
+{
+	"comments": {
+		"lineComment": "#",
+		"blockComment": [ "\"\"\"", "\"\"\"" ]
```

Comment:
Is there support for having more than one pair of open/close sequence, such that we can maintain support for the Java-style `/*...*/` comments?

Reply:
I don't know... probably... but I hope we drop them anyway, so I would not want to investigate this too much until we reach an agreement ;)